src/app/shared/modules/sai-editor/sai-editor.component.ts

```diff
@@ -83,6 +89,10 @@ export class SaiEditorComponent extends MultiRowEditorComponent<SmartScripts> im
   public readonly TARGET_PARAM2_TOOLTIPS  = SAI_TARGET_PARAM2_TOOLTIPS;
   public readonly TARGET_PARAM3_TOOLTIPS  = SAI_TARGET_PARAM3_TOOLTIPS;
   public readonly TARGET_PARAM4_TOOLTIPS  = SAI_TARGET_PARAM4_TOOLTIPS;
+  public readonly TARGET_X_TOOLTIPS  = SAI_TARGET_X_TOOLTIPS;
+  public readonly TARGET_Y_TOOLTIPS  = SAI_TARGET_Y_TOOLTIPS;
+  public readonly TARGET_Z_TOOLTIPS  = SAI_TARGET_Z_TOOLTIPS;
+  public readonly TARGET_O_TOOLTIPS  = SAI_TARGET_O_TOOLTIPS;
```

Comment:
@Kitzunu 2) I import them in the Component so I can use them in the Component's HTML